build/utils.py

```diff
@@ -27,6 +27,24 @@
 # Local
 from tuning.config import configs, peft_config, tracker_configs
 
+# The USER_ERROR_EXIT_CODE will be thrown when the process must exit
+# as result of a user input error. User-related errors should be
+# >= 1 and <=127 due to how some kubernetes operators interpret them.
+USER_ERROR_EXIT_CODE = 1
+# The INTERNAL_ERROR_EXIT_CODE will be thrown when training
+# abnormally terminates, and it is not clearly fault of the user.
+# System-level errors should be >= 128 and <= 254
+INTERNAL_ERROR_EXIT_CODE = 203
```

Comment:
Gaurav had left this comment in [slack](https://ibm-research.slack.com/archives/C06MZPQ4ENL/p1715432934933899?thread_ts=1715281892.534329&cid=C06MZPQ4ENL)
> Failing a training because system failed, be it because training itself failed, model copying failed or logs failed should always be a 5xx class error, which means user will not get billed

Should these exit codes be 500s then..?

Reply:
Since they are linux exit codes, they have to be between 0 and 255.

Reply:
Gotcha thanks 👍